testsuite/features/secondary/srv_salt_git_pillar.feature

```diff
@@ -0,0 +1,29 @@
+# Copyright (c) 2024 SUSE LLC
+# Licensed under the terms of the MIT license.
+
+@skip_if_github_validation
+@sle_minion
+@scope_salt
+Feature: Salt Master integration with Git Pillar
+
+  Scenario: Preparing Git Pillar configuration for Salt Master
+    When I setup a git_pillar environment on the Salt Master
+    And I wait for "30" seconds
+    Then file "/etc/salt/master.d/zz-testing-gitpillar.conf" should exist on server
```

Comment:
```suggestion
    And I wait at most 40 seconds until file "/etc/salt/master.d/zz-testing-gitpillar.conf" exists on "server"
```

When(/^I wait at most (\d+) seconds until file "([^"]*)" exists on "([^"]*)"$/) do |seconds, file, host|
  node = get_target(host)
  repeat_until_timeout(timeout: seconds.to_i) do
    break if file_exists?(node, file)

    sleep(1)
  end
end

Reply:
See explanation here: https://github.com/uyuni-project/uyuni/pull/9313#discussion_r1795250845

Reply:
The point is to avoid the passive wait of 30 seconds.

Reply:
Fixed.